operator/pkg/controllers/etcd/defaultspec.go

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
typically this logic would go in /pkg/apis/resourcename_defaults.go
https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/v1beta1/cluster_task_defaults.go